docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1778,10 +1810,180 @@ And then in the YAML file for your Kubernetes deployment, specify the environmen
                     key: binding
                     name: binding-<event_streams_service_instance_name>
 ```
+
 Please refer to https://cloud.ibm.com/docs/containers?topic=containers-service-binding for detailed instructions if needed.
 
+#### 4.11.3.2 fhir-server-config.json Configuration of audit logging service
+
+```
+"audit": {
+    "serviceClassName" : "com.ibm.fhir.audit.logging.impl.WhcAuditCadfLogService",
+    "serviceProperties" : {
+        "load": "config",
+        "mapper": "cadf",
+        "auditTopic": "FHIR_AUDIT",
+        "geoCity": "Dallas",
+        "geoState": "TX",
+        "geoCounty": "US",
+        "kafka" : {
+            "sasl.jaas.config": "********",
+            "bootstrap.servers": "********",
+            "sasl.mechanism": "PLAIN",
+            "security.protocol": "SASL_SSL",
+            "ssl.protocol": "TLSv1.2",
+            "ssl.enabled.protocols": "TLSv1.2",
+            "ssl.endpoint.identification.algorithm": "HTTPS"
+        },
+        "kafkaServers": "********",
+        "kafkaApiKey": "********"
+    }
+```
+
+The service may be map to the CADF format or the FHIR AuditEvent resource format by declaring a mapper type - 'cadf' or 'auditevent'. 
```

Comment:
I would have assumed the AuditEvent mapper would just be a different class.  If WhcAuditCadfLogService is used to map to both CADF and FHIR, then I think it should get renamed.

Reply:
Discussed as a team on code review